go-controller/pkg/ovn/secondary_layer3_network_controller_test.go

```diff
@@ -1004,3 +1067,84 @@ func buildNamespacedPortGroup(namespace, controller string) *nbdb.PortGroup {
 	pg.UUID = pg.Name + "-UUID"
 	return pg
 }
+
+func createHairpinAllowACLs(controllerName string) (*nbdb.ACL, *nbdb.ACL) {
```

Comment:
nit: the utils here, are we sure we cannot reuse the ones in policy_test.go ? some of these below seem to exist there as well right? trying to make sure there is no code duplication here by adding utils in each unit test file if there is a chance of reuse

Reply:
oh! yes @tssurya , i didn't notice `policy_test.go` before, my bad. reusuing most of its functions for multi homing test now.